.github/workflows/tests.yml

```
@@ -0,0 +1,41 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Read Bun Version from package.json
+        id: bun_version
+        run: echo "BUN_VERSION=$(node -p "require('./package.json').devDependencies['@types/bun'].replace('^', '')")" >> $GITHUB_ENV
+
+      - name: Cache Bun Installation
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)